detox/e2e/custom_sequencer.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const Sequencer = require('@jest/test-sequencer').default;
```

Comment:
This works fine without explicit install but would be nice to include in `devDependencies`. But I'll leave it to you.

Reply:
Added